File: tests/fixtures/extractor_corpus/sample.swift
import Foundation

class Greeter {
    let name: String

    init(name: String) {
        self.name = name
    }

    func greet() -> String {
        return "hi " + name
    }
}

func topLevel(first: Int, second: Int) -> Int {
    return first + second
}

struct Point {
    var x: Int
    var y: Int
}

protocol Runnable {
    func run()
}

File: tests/fixtures/extractor_corpus/sample.kt
package sample

import kotlin.math.max
import sample.store.*

class Greeter(val name: String) {
    fun greet(): String {
        return "hi " + name
    }
}

fun topLevel(first: Int, second: Int = 1): Int = max(first, second)

object Singletons {
    const val VERSION = 1
}

enum class Direction {
    NORTH,
    SOUTH
}

typealias Name = String
